### DISCUSSION TITLE
Task - Angular - Creating Child Components and Interpolation

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/spring-framework#25] [Tasks]
Create an Angular app that has two child components called `name-display` and `age-display`.

The `name-display` component shows your _firstname_ and _lastname_. The data for _firstname_ and _lastname_ will be sent from the component class (.ts file of the component).

The `age-display` component shows your _age_. The data for _age_ will be sent from the component class.

The `app-root` component will display "My First Angular Task".

**Comment by shreiyarandive** [comment]
https://github.com/shreiyarandive/wiley-angular-task1

**Comment by Sarfinaa** [comment]
https://github.com/Sarfinaa/Angular/tree/master/src

**Comment by Shubhi27** [comment]
https://github.com/Shubhi27/angular-tasks/tree/main/src

**Comment by preeti-max** [comment]
https://github.com/preeti-max/app.git

**Comment by mkaifahmed** [comment]
https://github.com/mkaifahmed/firstangular

**Comment by lalit-ky** [comment]
https://github.com/009lalitkumar/wiley-task1-angular

**Comment by Mrafe** [comment]
<a href="https://github.com/Mrafe/ang">code</a>

**Comment by Ashwesh09** [comment]
[Submitted](https://github.com/Ashwesh09/frontend/tree/main/my-app)

**Comment by rakeshGit419** [comment]
[https://github.com/rakeshGit419/firstangularpro.git](url)

**Comment by SrideviYamala** [comment]
https://github.com/SrideviYamala/AngularDemo

**Comment by ratnadeepikavuddagiri** [comment]
https://github.com/ratnadeepikavuddagiri/Angular

**Comment by Gauravgarje** [comment]
https://github.com/Gauravgarje/Wiley-Angular-Task001

**Comment by Harshita-Joshi** [comment]
https://github.com/Harshita-Joshi/Angular/tree/master

**Comment by KaminiJha** [comment]
## root component
```html
Demo app
<app-name-display></app-name-display>
<app-age-display></app-age-display>
```
## age component
```html
<p>age-display works!</p>
<h5>age is {{age}}</h5>
```
## name component
```html
<p>name-display works!</p>
<h2>first name is {{fname}} and last name is {{lname}}</h2>
```
